shared/unlock-folders/device-list.desktop.js

```diff
@@ -0,0 +1,80 @@
+// @flow
+
+import React, {Component} from 'react'
+import {globalStyles, globalColors} from '../styles/style-guide'
+import {Text, Button, Icon} from '../common-adapters'
+
+import type {Device} from '../constants/unlock-folders'
+
+export type Props = {
+  devices: ?Array<Device>,
+  toPaperKeyInput: () => void
+}
+
+class DeviceRow extends Component<void, {device: Device}, void> {
+  render () {
+    const isDesktop = this.props.device.type === 'desktop'
+    return (
+      <div style={{...globalStyles.flexBoxRow, marginBottom: 16}}>
+        <div style={{width: 24, marginLeft: 33, display: 'flex', justifyContent: 'center'}}>
+          <Icon type={isDesktop ? 'computer-big' : 'phone-big'} style={{height: 21}}/>
+        </div>
+        <Text type='BodySemibold' style={{...globalStyles.italic, marginLeft: 16, cursor: 'default'}}>{this.props.device.name}</Text>
```

Comment:
this specific italic style, i don't think we'll italicize any other types, i think it'll be just this right @cecileboucheron